contrib_observer.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//           (c) 2017 IBM Corp.
```

Comment:
When you sign the CLA it assigns copyright to Uber, so please don't include this here, because it restricts our ability to manage licensing of Jaeger packages.